### PR TITLE
Concert opts

### PR DESCRIPTION
**Pull request by mitchjacksontech**
Please review this, @casey-collab

### CODE REVIEW

File: english/music-festival/lollapalooza-1997/lollapalooza-1997.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+witness performances by:
+
+Orbital,
+Devo,
+The Prodigy,
+The Orb,
+Snoop Dog with Tupac,
+and the closing act, Tool
+
+Afterwards, you find yourself [holding the door](../hold_the_door/hodor.md)
```
Comment (casey-collab) [inline english/music-festival/lollapalooza-1997/lollapalooza-1997.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/music-festival/music-festival.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Welcome to the heart of your most fondly remembered music festival.
+You don't need a ticket.  You've been here before.
+
+Where are you?
+
+[Woodstock 1994](woodstock-1994.md)
```
Comment (casey-collab) [inline english/music-festival/music-festival.md]: This link is also invalid.

File: english/music-festival/music-festival.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+[Lollapalooza 1997](lollapalooza-1997.md)
```
Comment (casey-collab) [inline english/music-festival/music-festival.md]: This link is also invalid.

File: english/music-festival/woodstock-1994/woodstock-1994.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+You're trapped in a memory of Woodstock '94.  Were these your experiences,
+or do they belong to someone else... perhaps [many others](https://www.youtube.com/watch?v=Cz227EU_1mU)?
+
+You slide in the mud while being entertained by:
+[Blind Melon](https://www.youtube.com/watch?v=3xph8pPJMB8)
+[Primus](https://www.youtube.com/watch?v=v-HUK8qR9QA)
+[Nine Inch Nails](https://www.youtube.com/watch?v=3lbFfiLAj_A)
+
+Before your favorite band can play, you are
+back in [the forest](../forest/forest.md) again.
```
Comment (casey-collab) [inline english/music-festival/woodstock-1994/woodstock-1994.md]: This link is also invalid.

File: english/music-festival/woodstock-1994/woodstock-1994.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+You're trapped in a memory of Woodstock '94.  Were these your experiences,
+or do they belong to someone else... perhaps [many others](https://www.youtube.com/watch?v=Cz227EU_1mU)?
+
+You slide in the mud while being entertained by:
+[Blind Melon](https://www.youtube.com/watch?v=3xph8pPJMB8)
+[Primus](https://www.youtube.com/watch?v=v-HUK8qR9QA)
```
Comment (casey-collab) [inline english/music-festival/woodstock-1994/woodstock-1994.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: english/music-festival/woodstock-1994/woodstock-1994.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+[Nine Inch Nails](https://www.youtube.com/watch?v=3lbFfiLAj_A)
```
Comment (casey-collab) [inline english/music-festival/woodstock-1994/woodstock-1994.md]: This link is also on the line directly after another link.

File: english/music-festival/woodstock-1994/woodstock-1994.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+You're trapped in a memory of Woodstock '94.  Were these your experiences,
```
Comment (casey-collab) [inline english/music-festival/woodstock-1994/woodstock-1994.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/music-festival/lollapalooza-1997/lollapalooza-1997.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
+You leave work at Bell's Amusement Park in Tulsa OK and
```
Comment (casey-collab) [inline english/music-festival/lollapalooza-1997/lollapalooza-1997.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/music-festival/lollapalooza-1997/lollapalooza-1997.md (hunk 1 above)
Comment (mitchjacksontech) [inline english/music-festival/lollapalooza-1997/lollapalooza-1997.md]: Please review this, @casey-collab

**Pull request closed without merging**